src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,39 @@
 import React from 'react';
 
-export const TodoFilter: React.FC = () => {
+import { Status } from '../../types/Status';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions } from '../../features/filter';
+
+export const TodoFilter = () => {
+  const dispatch = useAppDispatch();
+  const filter = useAppSelector(state => state.filter);
+
+  const handleGlobalFilterChange = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ) => {
+    dispatch(actions.changeGlobal(event.target.value as Status));
+  };
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    dispatch(actions.changeQuery(event.target.value));
```

Comment:
```suggestion
    dispatch(actions.changeQuery(event.target.value.trim()));
```
for fix this comment
https://github.com/mate-academy/react_redux-list-of-todos/pull/678#pullrequestreview-1702295353